Intersect (Core)/Network/Packets/Server/ItemProperties.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Intersect.Enums;
+using Intersect.GameObjects;
+using Intersect.Utilities;
+using MessagePack;
+
+namespace Intersect.Network.Packets.Server
+{
+    [MessagePackObject]
+    public partial class ItemProperties
+    {
+        public ItemProperties()
+        {
+        }
+
+        public ItemProperties(Guid itemBaseId)
+        {
+            var descriptor = ItemBase.Get(itemBaseId);
+            if (descriptor == null)
+            {
+                return;
+            }
+
+            if (descriptor.ItemType != ItemTypes.Equipment)
+            {
+                return;
+            }
+
+            for (var i = 0; i < (int)Stats.StatCount; i++)
+            {
+                // TODO: What the fuck?
+                RandomStats[i] = Randomization.Next(-descriptor.StatGrowth, descriptor.StatGrowth + 1);
+            }
+        }
+
+        [Key(1)]
+        public int[] RandomStats { get; set; } = new int[(int)Stats.StatCount];
```

Comment:
This should be called `StatModifiers` instead of `RandomStats`. Their _generation_ might be random, but that's not the only way to populate them.